decksite/controllers/api.py

```diff
@@ -40,32 +40,40 @@
     'name': fields.String(),
 })
 
-DECK = APP.api.model('Deck', {
-    'id': fields.Integer(readonly=True),
-    'name': fields.String(),
-    'created_date': fields.DateTime(),
-    'updated_date': fields.DateTime(),
-    'wins': fields.Integer(),
-    'losses': fields.Integer(),
-    'draws': fields.Integer(),
-    'finish': fields.Integer(),
-    'archetype_id': fields.Integer(),
-    'archetype_name': fields.String(),
-    'source_url': fields.String(),
-    'competition_id': fields.Integer(),
-    'competition_name': fields.String(),
-    'person': fields.String(),
-    'decklist_hash': fields.String(),
-    'retired': fields.Boolean(),
-    'colors': fields.List(fields.String()),
-    'omw': fields.Integer(),
-    'season_id': fields.Integer(),
-    'maindeck': fields.List(fields.Nested(DECK_ENTRY)),
-    'sideboard': fields.List(fields.Nested(DECK_ENTRY)),
-    'url': fields.String(),
-    'source_name': fields.String(),
-    'competition_type_name': fields.String(),
-})
+def get_deck_model(include_last_update: bool) -> Any:
```

Comment:
How expensive is including last update? It might be nice just to always send it to avoid this boolean and extra variables like DECK_WITH_ARCH_CHANGE etc. Failing that how about we send it but send it empty?

Reply:
It's slightly expensive because you need to join and find a MAX on it. It doesn't seem to create a huge overhead though, so I could add it to the decks requests as well. I could also just use the same model, but send nulls when loading from /api/decks. That last approach seems the best